ingest/src/main.rs

```diff
@@ -51,9 +51,12 @@ impl Server {
         poc_metrics::start_metrics(&settings.metrics)?;
 
         let (shutdown_trigger, shutdown_listener) = triggered::trigger();
+        let mut sigterm = signal::unix::signal(signal::unix::SignalKind::terminate())?;
```

Comment:
does this need to be called outside of the spawned task for any reason? just to catch and propagate the error if it occurs?

Reply:
just to propagate the error, rather than unwrapping in the tokio::spawn call.